docs/components/tools.md

```diff
@@ -815,7 +815,7 @@ Flags:
       --rewrite.to-relabel-config-file=<file-path>
                                 Path to YAML file that contains relabel configs
                                 that will be applied to blocks
-      --tmp.dir="/tmp/thanos-rewrite"
+      --tmp.dir="/var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/thanos-rewrite"
```

Comment:
Seems like a random change :smile:

Reply:
For some reason `make docs` does this on my machine. Should be reverted now :)